tests/spread/integration/file/task.yaml

```diff
@@ -0,0 +1,13 @@
+summary: Integration tests for file
+
+execute: |
+  rootfs="$(install-slices file_bins)"
+
+  chroot "${rootfs}/" file --help
+  echo "Hello World" > "${rootfs}/testfile" 
+  output=$(chroot "${rootfs}/" \
+    file -b testfile)
+  if [[ "$output" != "ASCII text" ]]; \
```

Comment:
`[[ ... ]]` is a bash-only construct, so we should just use `[ ... ]` instead since the integration tests use dash.

Reply:
Also, the typical syntax I use is:
```shell
if [ ... ]; then
  ...
fi
```

I.e., put the `then` statement on the same line as the `if` statement.
And there shouldn't be any need to use continuation markers (`\`) for it.